rvm.c

```diff
@@ -33,7 +57,7 @@ static bool recover_blocks(rvm_cfg_t *cfg)
             continue; //Freed memory
 
         /* Allocate local storage for recovered block */
-        err = posix_memalign(&(blk->local_addr), cfg->blk_sz, blk->size);
+        err = posix_memalign(&(blk->local_addr), cfg->blk_sz, cfg->blk_sz);
```

Comment:
FYI: This had to change because we can't protect less than a page. Allocating less than a full page allows malloc() to use that space, this means that when we protect the allocation, we get segfaults for normal (non rvm) accesses. Long-term we just need to write a real allocator.